Packs/CortexResponseAndRemediation/Playbooks/silent-Abnormal_first_access_to_a_resource_via_SSO_in_the_organization.yml

```diff
@@ -0,0 +1,5203 @@
+      name: Preliminary Filter
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "Yes":
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: "Yes"
+      condition:
+      - - operator: isEqualNumber
+          left:
+            value:
+              simple: Core.OriginalAlert.event.factory_anomaly_score
+            iscontext: true
+          right:
+            value:
+              simple: "0"
+        - operator: isEmpty
+          left:
+            value:
+              complex:
+                root: Core.OriginalAlert.event.auth_server
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: Core.OriginalAlert.event.auth_server
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Azure
+                    ignorecase: true
+                  - operator: isEqualString
+                    left:
+                      value:
+                        simple: Core.OriginalAlert.event.auth_server
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Okta
+                    ignorecase: true
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 476
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: f42fdc47-5f88-49a0-b8ae-407c6ac858a9
+    type: title
+    task:
+      id: f42fdc47-5f88-49a0-b8ae-407c6ac858a9
+      version: -1
+      name: Check FP
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 331
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: a92686a8-3fa9-49c2-8c15-e28c6304528d
+    type: regular
+    task:
+      id: a92686a8-3fa9-49c2-8c15-e28c6304528d
+      version: -1
+      name: Get alert frequency
+      description: |-
+        Searches Demisto alerts. A summarized version of this scrips is available with the summarizedversion argument.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Permission-Management
```

Comment:
Need to change this from line 229 -233. If this is for XSIAM need to change Demisto alerts to Cortex XSIAM alerts. Also there is no default limited user role in XSIAM.